packages/app/cypress/e2e/studio/studio-new-tests.cy.ts

```diff
@@ -128,13 +128,13 @@ describe('studio functionality', () => {
   it('creates a new test for a specific suite with the url already defined', () => {
     launchStudio({ specName: 'spec-w-visit.cy.js', createNewTestFromSuite: true })
 
-    // create a new test from a specific suite
-    cy.findByTestId('create-new-test-from-suite').click()
-
-    inputNewTestName()
+    inputNewTestName({ creatingNewTestFromWelcomeScreen: false })
 
     cy.percySnapshot()
 
+    // make sure that the visit has run and we're recording studio commands
+    cy.get('[data-cy="record-button-recording"]').should('be.visible')
```

Comment:
Suggest taking the snapshot after we know we're ready.
```suggestion
    // make sure that the visit has run and we're recording studio commands
    cy.get('[data-cy="record-button-recording"]').should('be.visible')

    cy.percySnapshot()
```

Reply:
yeah this makes more sense